test/xpu_api/numerics/numeric.ops/numeric.ops.gcd/xpu_gcd.pass.cpp

```diff
@@ -0,0 +1,150 @@
+// -*- C++ -*-
+//===-- xpu_gcd.pass.cpp --------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include <iostream>
+
+#include <CL/sycl.hpp>
+#include <oneapi/dpl/numeric>
```

Comment:
oneDPL headers should be before other headers to avoid issues with libstdc++9, libstdc++10

Reply:
done.